docs/customising/config.md

```diff
@@ -994,6 +995,22 @@ SHARED_DIRECTORIES:
   </dd>
 
   <dt>
+    <a name="widget_page"><code>WIDGET_PAGE</code></a>
+  </dt>
+  <dd>
+     If you would like to give users the opportunity to insert HTML 'widgets' into their other websites, advertising the
+     requests they've made in Alaveteli, set this to true.
```

Comment:
Should add a link to some documentation on adding a widget. Just reading this doesn't give me a sense of what a widget looks like and whether its something I'd want enabled.